sig-pm/SIG PM representatives.md

```diff
@@ -0,0 +1,26 @@
+|  **SIG** | **Name** | **GitHub ID** | **e-mail** | **Secondary** |
+|  ------ | ------ | ------ | ------ | ------ |
+|  API machinery |  |  |  |  |
+|  Apps | Dan Paik, Antoine Legrand | danpaik; ant31 | danpaik@google.com | antoine.legrand@coreos.com |
+|  Auth | CJ Cullen | cjcullen | cjcullen@google.com |  |
+|  Autoscaling | Solly Ross | directxman12 | sross@redhat.com |  |
+|  AWS | Justin Santa Barbara | justinsb |  |  |
+|  CLI | Joe Beda | jbeda | joe@bedafamily.com |  |
+|  Cluster Lifecycle | Dan Gillespie |  |  |  |
+|  Cluster Ops | Jason Singer DuMars | jdumars | jdumars@deis.com | Rob Hirschfeld |
+|  Contrib Experience |  |  |  |  |
+|  Docs |  |  |  |  |
+|  Federation | David Aronchick |  | aronchick@google.com |  |
+|  Instrumentation | Piotr Szczesniak | piosz | pszczesniak@google.com | Patrick Christopher (pat.christopher@gmail.com) |
+|  Network |  |  |  |  |
+|  Node | Caleb Miles | calebamiles | caleb.miles@coreos.com |  |
```

Comment:
Mind add `PengfeiNi | feiskyer | feiskyer@hyper.sh` for CRI (seems no this sig?) or Node here?